### ISSUE TITLE
Not Really an Issue

### ISSUE THREAD

**Issue by mva1985** [blizzard4591/openMittsu#52]
Just wondering if there is a roadmap for the development of this application?

Also, just wondering if this will ever be able to support file sharing?

And I just wanted to thank you for always providing a zip file so it can be run portable...  thanks so much!!

**Comment by blizzard4591** [comment]
Thank you for your kind comment :)

Regarding the roadmap: I planned to support the Threema Safe Backup things, this is almost completely implemented, but I did not find the time nor motivation to finish it... 
Also, the communication to the Threema developers has dried up.
What pushes me to invest time is seeing people actually use the software and requesting features. But that also has not happened in a while ;)

**Comment by hkramski** [comment]
Requesting features? Easy:
I would like to have a cmd line option to provide the database password (or a file containing the password) and another option to start openMittsu in (auto-)connected mode. Maybe also the database path as a parameter.
Thanks for your good work, I really use and like it a lot.

**Comment by rugk** [comment]
Well… I guess the proper way would be to open new issues for new feature requests. :smile:

**Comment by hkramski** [comment]
> Well… I guess the proper way would be to open new issues for new feature requests. smile

Yes. that's true, just filed https://github.com/blizzard4591/openMittsu/issues/53.
